utils/fsnotify.go

```diff
@@ -0,0 +1,70 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/cri-o/cri-o/internal/log"
+	"github.com/fsnotify/fsnotify"
+)
+
+func Watch(ctx context.Context, path string, opsToWatch []fsnotify.Op) chan error {
```

Comment:
Do you think it makes sense to add some unit test cases for this function?

Reply:
it does!